### PR TITLE
contain the long list inside a div which is 50% of vertical height

### PR DESCRIPTION
**Pull request by pmario**
contain the long list inside a div which is 50% of vertical height

### CODE REVIEW

**Comment by Jermolene** [comment]
Hi @pmario I agree that the current listing is very unwieldy with 2400-odd entries. However, this change means that  with some browser/OS combinations one can no longer see that the list is truncated or scrollable:

image

This is Chrome on macOS.

On balance, I think I'd favour removing the list entirely. The rest of the tiddler could do with improvement, but the list really doesn't help.

**Comment by pmario** [comment]
The new commit only shows the modified shadows and creates a button to "open" the shadows in the right sidebar.

**Comment by pmario** [comment]
looks like this:

grafik

**Comment by pmario** [comment]
@Jermolene bump. See new screenshot

**Comment by pmario** [comment]
@Jermolene bump.

**Comment by pmario** [comment]
@Jermolene .. this is a docs change only

File: editions/tw5.com/tiddlers/concepts/ShadowTiddlers.tid [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 
-The current shadow tiddlers are:
+!! Overridden Shadow Tiddlers
 
-<$list filter="[all[shadows]sort[title]]"/>
+<<list-links "[is[tiddler]is[shadow]sort[title]]">>
+
+!! Shadow Tiddlers
+
+Shadow tiddlers can be found in the <<click "Right Sidebar : More : Shadows tab">>
```
Comment (Jermolene) [inline editions/tw5.com/tiddlers/concepts/ShadowTiddlers.tid:19]: Apologies, more consistency issues:

* The docs generally refer to the "sidebar" not the "right sidebar"
* The docs frequently refer to a specific sidebar tab without having a button to activate that tab. One might argue that this particular docs tiddler is enhanced by the button, but I think that consistency is more important. If we think that this is a worthwhile improvement then it needs to be implemented consistently across all the docs

I'd recommend removing the macros and changing the last line to:

```
A full list of shadow tiddlers can be found in the sidebar in the "More" -> "Shadows" tab
```

**Review by Jermolene — COMMENTED** [review]
Thanks @pmario

**Pull request merged**